iox_client_test.go

```diff
@@ -19,24 +19,42 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
+// Return the environment value for env, or default to the provided fallback
+// value.
+func envOrDefault(env string, fallback string) string {
+	v, ok := os.LookupEnv(env)
+	if !ok {
+		return fallback
+	}
+	return v
+}
+
+// Return the hostname of the test IOx instance.
+func getTestHost() string {
+	return envOrDefault("INFLUXDB_IOX_HOST", "localhost")
+}
+
+// Return the HTTP port for the test IOx instance.
+func getTestHttpPort() string {
+	return envOrDefault("INFLUXDB_IOX_HTTP_PORT", "8080")
+}
+
+// Return the gRPC port for the test IOx instance.
+func getTestGRPCPort() string {
+	return envOrDefault("INFLUXDB_IOX_GRPC_PORT", "8082")
+}
```

Comment:
This change lets various helper funcs load the same values by sharing the same fallback value logic.